src/skottie-player.ts

```diff
@@ -3,6 +3,53 @@
  */
 
 import {LitElement, html, customElement, property, css} from 'lit-element';
+const LottieModule = (window as any).lottie;
+import * as lottie from 'lottie-web/build/player/lottie_light';  // Using light version for lighter bundle
+type LottiePlayerRoot = typeof lottie.default;
+const lottiePlayer = LottieModule as any as LottiePlayerRoot;
+console.log(lottie);  
+console.log(lottiePlayer);
+
+interface layer {
+  ty?: Number,
+  ks?: any, // transform object
```

Comment:
Nice comment. The other acronyms are also hard to guess their meanings. Consider adding comments for all the acronyms.